proxy/plan/plan_delete_test.go

```diff
@@ -59,6 +59,15 @@ func TestMycatShardDeleteWithWhere(t *testing.T) {
 				},
 			},
 		},
+		{ // column should be case insensitive
+			db:  "db_mycat",
+			sql: "delete from tbl_mycat where ID = 0",
```

Comment:
Is it need to fix case sensitive problems in namespace (database, table name and sharding column name) ?